src/commands/import.ts

```diff
@@ -87,8 +107,24 @@ export default class Import extends Kommand {
         const total = await restoreProfiles(this, dump)
 
         this.logOk(`[profiles] Imported ${total} profiles`)
+      } catch (error) {
+        this.logKo(`Error during import of ${file}: ${error.message}. Skipped.`)
       }
-      catch (error) {
+    }
+    if (files.userMapping.length !== 0) {
+      const file = files.userMapping[0]
+      try {
+        this.logInfo(`[user mapping] Start importing user mapping in ${file}`)
+        const dump = JSON.parse(fs.readFileSync(file, 'utf8'))
+
+        const mapping = dump.content.mapping
+        delete mapping.profileIds
```

Comment:
Why deleting `profileIds`? They are part of the mappings (even if I agree that yes they should already exists still they are part of the default mappings)

Reply:
I just followed the example of https://docs.kuzzle.io/sdk/js/7/controllers/security/update-user-mapping/ wich doesn't add profileIds file in the body.
Should i let this field ?

Reply:
I would say yes, this is harmless and also IMHO it's a good thing to not hide this from users